float64 raw_timestamp
float64 raw_requested_velocity
float64 raw_velocity
float64 raw_current
float64 raw_temperature
float64 raw_supply_voltage
float64 raw_pwm
float64 raw_torque
float64 raw_acceleration
